react-components/src/apps/admin/AllStats/AllStats.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * @author Flynn
+ *
+ * Description: This file is importing all the necessary files & everything is being rendered here.
+ * Files: All the "Tab" files are imported in NavBarTabs under the TabPanels.
+ *        All the "Table" files are imported under their respective "Tab" files.
+ *        The data files for storybook are stored in the dataFiles folder
+ *        All the components used in AllStats.js are stored in the AppsStatsComponents folder
+ */
+
+import React, { Component } from "react";
+import DatePicker from "./AppStatsComponents/datePicker.js";
+import "./AppStatsComponents/button";
+import ColorButton from "./AppStatsComponents/button";
+import NavBarTabs from "./AppStatsComponents/NavBarTabs";
+import ids from "./AppStatsComponents/AllStatsIDs";
+import { withStyles } from "@material-ui/core";
+import styles from "./AllStatsStyle.js";
+import { getMessage, withI18N } from "@cyverse-de/ui-lib";
+import myMessagesFile from "./AppStatsComponents/messages.js";
+import build from "@cyverse-de/ui-lib/src/util/DebugIDUtil";
+
+class AllStats extends Component {
+    render() {
+        const { classes } = this.props;
+
+        return (
+            <div id={ids.MAIN_PAGE}>
+                <header>
+                    <div className={classes.datePickers}>
+                        <DatePicker
+                            label={getMessage("startDate")}
+                            id={build(ids.MAIN_PAGE, ids.START_DATE)}
+                            className={classes.datePicker}
+                        />
+                        <DatePicker
+                            label={getMessage("endDate")}
+                            id={build(ids.MAIN_PAGE, ids.END_DATE)}
+                            className={classes.datePicker}
+                        />
+                        <ColorButton
```

Comment:
Since this button uses the `primary` color (which is CyVerse blue), I don't think you need to use `ColorButton` at all, which seems to be trying to apply purple colors?  I think you could just stick with the regular material-ui `TextButton` and remove the ColorButton.js file.